contracts/VToken.sol

```diff
@@ -85,7 +89,8 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         uint8 decimals_,
         address payable admin_,
         AccessControlManager accessControlManager_,
-        VTokenInterface.RiskManagementInit memory riskManagement
+        VTokenInterface.RiskManagementInit memory riskManagement,
+        StableRateModel stableRateModel
```

Comment:
Items in this contract should be ordered per spec

Reply:
Fixed, can check the changes at [this commit](2981e0d08b247b7c7a7dc3461e9701d7d593c520).